tailwind.config.js

```diff
@@ -1,5 +1,7 @@
+const colors = require('tailwindcss/colors')
+
 module.exports = {
-  theme: [],
+  theme: { extend: { colors: { gray: colors.blueGray, orange: colors.orange, blue: colors.lightBlue } } },
```

Comment:
Where the `orange` and `blue` colors added because they were found? Also, is the `blueGray` the default or based on closest match to Tailwind 1.0 colors?

Finally, did we verify these belong in `theme.extend`? The docs show [curating colors](https://tailwindcss.com/docs/customizing-colors#curating-colors) being done under `theme.colors`.

Reply:
Yeah, blue and orange color classes are referenced in the code base so that's why it included them.

Orange isn't part of the defaults any longer, so it needs to be manually included when detected.

Blue doesn't strictly need to be included because it is there by default, but the default blue is quite different from Tailwind 1.0 and they now provide multiple blues to choose from, with `lightBlue` being the best match for Tailwind 1.0. Take a look at https://jessarcher.com/blog/tailwind-2-color-changes/ to see just how different it is.

Lastly, we could put them in `theme.colors` but then we would need to specify every color that is needed, rather than just [extending the defaults](https://tailwindcss.com/docs/customizing-colors#extending-the-defaults). This would make it harder to use things like Tailwind UI that might depend on a handful of extra colors (like alerts needing yellows, blues, oranges, etc).